### ISSUE TITLE
Green and Blue support colors do not pass color contrast test

### ISSUE THREAD

**Issue by carmacleod** [IBM/carbon-design-kit#75]
Please ensure that the support colors all pass the color contrast test for the suggested text color.
https://www.carbondesignsystem.com/guidelines/color/swatches#support-colors

The 2 failing support colors are white on green support-02 and white on blue support-04.
(The white on red support-01 and black on yellow support-03 pass).

1. White #ffffff text on green #5aa700 background is a contrast ratio of 3.02, which is less than the required ratio of at least 4.5 to pass WCAG 2.0 AA for accessibility.

The closest passing green color I could find is #458500.

2. White #ffffff text on blue #5aaafa background is a contrast ratio of 2.45, which is less than the required ratio of at least 4.5 to pass WCAG 2.0 AA for accessibility.

This color passes with black text, or you could try to find a passing blue.

**Comment by laurenmrice** [comment]
For inline notifications, since they are transparent on both the white and blue themes (and toasts on white), the text color is the only item that needs to pass accessibility by 4.5. An icon would be the same as well if it was the only indicator but it is a supporting factor.

**Comment by carmacleod** [comment]
I'm sorry, but I don't understand the explanation. Are you saying that authors have to figure out a passing text color if they use the support colors with text? The examples given do fail, so it is setting up authors for failure.
